doc/user/interop.md

```diff
@@ -100,6 +100,29 @@ objects.
 
 TruffleRuby ignores the `internal` flag to this message.
 
+### `KEY_INFO`
+
+If the object is a Ruby `Hash`, and the key is in the hash, `READABLE`,
+`WRITABLE` are set. If the key is not in the hash, neither are set. Note that
```

Comment:
WRITABLE should only be set if the object is not frozen.

Reply:
Fixed.